app/src/main/res/xml/credit_cards_preferences.xml

```diff
@@ -2,14 +2,14 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Sorry to bring this up again since I recall you said you were removing it. Did we need the androidx.preference still?

Reply:
I only removed it from `login_preferences.xml`. Sorry about that. Fixed now